src/app/tasks/task-status-selector/task-status-selector.component.html

```diff
@@ -0,0 +1,30 @@
+<p>task-status-selector works!</p>
+
+<ul [ngClass]="{'dropdown-menu' :inMenu}" class="task-status-selector">
+    <li [hidden]="!assessingUnitRole" class="col-xs-6 col-sm-4 col-md-4" *ngFor="let action of taskEngagementConfig.tutorTriggers" >
+      <div class="row">
+      <button class="col-xs-12 btn task-status {{action.class}}" [ngClass]="{'active' : project.selectedTask.status == action.status}" (click)="triggerTransition(action.status)" [(ngModel)]="project.selectedTask.status" btn-radio="action.status" uncheckable>
+        <i class="{{action.icon}} fa-2x task-status-icon"></i>
+        <strong>{{action.label}}</strong>
+      </button>
+      </div>
+    </li><!--/tutor-statuses-->
+    <li [hidden]="assessingUnitRole ||
+     taskService.rejectFutureStates[task.status].length == 0" class="col-xs-6 col-sm-4 col-md-4">
+      <div class="row">
+      <button class="col-xs-12 btn task-status {{task.statusClass()}} active" uncheckable>
+        <i class="{{task.statusIcon()}} fa-2x task-status-icon"></i>
+        <strong>{{task.statusLabel()}}</strong>
+      </button>
+      </div>
+    </li>
+    <li class="col-xs-6 col-sm-4 col-md-4" *ngFor="let action of task.filterFutureStates(studentTriggers)" > <!--taskEngagementConfig.studentTriggers-->
+      <div class="row">
+        <button class="col-xs-12 btn task-status {{action.class}}" [ngClass]="{'active' : project.selectedTask.status == action.status}" (click)="triggerTransition(action.status)" [(ngModel)]="project.selectedTask.status" btn-radio="action.status" >
+          <i class="{{action.icon}} fa-2x task-status-icon"></i>
+          <strong>{{action.label}}</strong>
+        </button>
+      </div>
+    </li><!--/student-statuses-->
+  </ul>
+  
```

Comment:
This still uses bootstrap styles and old icons, should be angular material mat-icon and styles